wit-packages/golem-rpc/wasm-rpc.wit

```diff
@@ -1,65 +0,0 @@
-package golem:rpc@0.1.0;
```

Comment:
These guys seem to be unused and are outdated.

Reply:
Put them back. Should they be bumped to the latest wit?

Reply:
No, I will do it when we release Golem 1.2